app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java

```diff
@@ -96,11 +96,14 @@ public void onChanged(@Nullable List<Notification> notifications) {
     /**
      * Starts BookInfoActivity
      */
-//    private void showBookInfo(Notification notification) {
-//        Intent showBookIntent = new Intent(getApplicationContext(), /* TODO: WAITING ON REFACTORED BOOKINFOACTIVITY */);
-//        showBookIntent.putExtra("bookID", notification.getBookID());
-//        startActivity(showBookIntent);
-//    }
+    private void showBookInfo(Notification notification) {
+        Intent showBookIntent = new Intent(getApplicationContext(), ShowRequestInfoActivity.class);
+        showBookIntent.putExtra("bookID", notification.getBookID());
+        // TODO: rStatus in ShowRequestInfoActivity should be obtained within itself
+        // BELOW IS PLACEHOLDER AND SHOULD BE THE STATUS OF THE REQUEST INSTEAD OF THE NOTIFICATION
+        showBookIntent.putExtra("rStatus", notification.getrNotificationType().toString());
```

Comment:
Might be good to make rStatus a static final constant in the target class(ShowRequestInfoActivity) and then reference that constant instead

Reply:
I'm not really sure about everything that's going on with the Activity currently. Do you think I should leave it now and refactor in the future or?

Reply:
I think just change it now. It should be fairly easy to change

Reply:
done